connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/DedicatedMirrorIntegrationTest.java

```diff
@@ -353,6 +353,9 @@ private <T extends SourceConnector> void awaitTaskConfigurations(MirrorMaker mm,
                         .map(TaskInfo::config)
                         .allMatch(predicate);
             } catch (Exception ex) {
+                if (ex instanceof RebalanceNeededException) {
+                    throw ex;
+                }
```

Comment:
Can we add a comment here explaining this non-obvious behavior?
```suggestion
                if (ex instanceof RebalanceNeededException) {
                    // It's okay to retry on this error; happens when a worker has read a new config
                    // from the config topic but hasn't completed the subsequent rebalance yet
                    throw ex;
                }
```

Reply:
you are right, let me add the comment, thanks!